network/src/peer/mod.rs

```diff
@@ -46,14 +46,22 @@ mod test;
 #[cfg(any(test, feature = "fuzzing"))]
 pub mod fuzzing;
 
+/// Requests [`Peer`] receives from the `PeerManager`.
 #[derive(Debug)]
 pub enum PeerRequest {
-    // Temporary message types until we can remove NetworkProvider and use the
-    // diem_channel::Receiver<ProtocolId, NetworkRequest> directly.
-    SendDirectSend(Message),
+    /// Send an RPC request to peer.
     SendRpc(OutboundRpcRequest),
+    /// Fire-and-forget style message send to peer.
+    SendMessage(Message),
```

Comment:
Any reason to rename DirectSend to just message?

Reply:
`SendDirectSend` is definitely clearer, i'll rename : )